ui_framework/src/components/table/_index.scss

```diff
@@ -0,0 +1,12 @@
+$kuiTableCellPadding: $kuiSize;
```

Comment:
It's actually lower than this in the mocks, otherwise it's too airy. `$kuiSize - $kuiSizeXS`